bubble_sort/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+ 
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -5,7 +5,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```